.github/workflows/build-runner-images.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Runner EC2 Images
+
+on:
+  schedule:
+    # 00:00 on Monday each week
+    - cron: "0 0 * * 1"
+    
+jobs:
+  build-image:
+    name: Build
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packer/github-actions
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v3
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_NUMBER }}:role/github-to-aws-oidc
```

Comment:
This can be a variable instead of a secret, and I would make it specific to the runners with something like `RUNNER_ACCOUNT`.